include/sempr/entity/RDFValue.hpp

```diff
@@ -148,6 +177,27 @@ class RDFValue {
         return this->get<std::string>() == other;
     }
 
+    bool operator != (const char* other) {
+        return !(this->operator ==(other));
```

Comment:
Do you no any options to avoid the special case for const char* ?

Reply:
Small addition: Of course, if you are willing to use `std::string` _all the time_ we won't need this. But this would mean that you'd have to write:
```c++
if (m["name"] == std::string("Max")) do_something();

// instead of

if (m["name"] == "Max") do_something();
```
If you ask me, I'm happy with the overload :D

Reply:
I expected an answer like this ^^ 
Its fine this way :+1: